src/pages/workspace/workflows/ToggleSettingsOptionRow.tsx

```diff
@@ -0,0 +1,85 @@
+import React, {useState} from 'react';
+import {View} from 'react-native';
+import type {ViewStyle} from 'react-native';
+import useStyleUtils from '@hooks/useStyleUtils';
+import type {SvgProps} from 'react-native-svg';
+import Icon from '@components/Icon';
+import Switch from '@components/Switch';
+import Text from '@components/Text';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import ImageSVG from '@components/ImageSVG';
+import Dot from '@assets/images/dot.svg';
+
+type ToggleSettingOptionRowProps = {
+    icon: React.FC<SvgProps>;
+    title: string;
+    subtitle: string;
+    hasBeenToggled: boolean;
+    onToggle: (isEnabled: boolean) => void;
+    subMenuItems?: React.ReactNode;
+    isEndOptionRow?: boolean;
+};
+
+function ToggleSettingOptionRow({icon, title, subtitle, onToggle, subMenuItems, isEndOptionRow, hasBeenToggled}: ToggleSettingOptionRowProps) {
+    const [isEnabled, setIsEnabled] = useState(hasBeenToggled);
+    const styles = useThemeStyles();
+    const StyleUtils = useStyleUtils();
+
+    const toggleSwitch = () => {
+        setIsEnabled(!isEnabled);
+        onToggle(!isEnabled);
+    };
+    const {isSmallScreenWidth} = useWindowDimensions();
+
+    function VerticalDots({count}: {count: number}) {
+        return (
+            <View style={StyleUtils.getWorkspaceWorkflowsDotStyle(isEnabled, isSmallScreenWidth) as ViewStyle}>
+                {Array.from({length: count}, () => (
+                    <ImageSVG
+                        contentFit="contain"
+                        src={Dot}
+                        height={isSmallScreenWidth ? 4 : 6}
+                        width={isSmallScreenWidth ? 4 : 6}
+                        pointerEvents="none"
+                    />
+                ))}
+            </View>
+        );
+    }
```

Comment:
We can't add it as the background on the Workflows section. We need to repeat the dots vertically on the Y axis.

We need to render it as a component, otherwise some properties will not be taken into account (backgroundSize for example). That was the reason dots weren't showing in iOS, we can't just use the same CSS properties as in Web.

Reply:
cc @Expensify/design could you please address @ishpaul777 [comment](https://github.com/Expensify/App/pull/36345#discussion_r1497294832) just above?

If we decide to omit the dots, it would simplify a lot the code and fix the current limitations we're facing due to React native.

Reply:
This is not relevant anymore as we'll remove the dots. See Shawn's comment [here](https://github.com/Expensify/App/pull/36345#issuecomment-1956588423)